src/components/PupilViews/PupilReview/PupilReview.view.tsx

```diff
@@ -144,13 +146,43 @@ export const PupilViewsReview = (props: PupilViewsReviewProps) => {
       })}`;
       navigator.clipboard.writeText(shareUrl);
       setIsAttemptingShare("shared");
+      if (sectionResults["starter-quiz"]?.isComplete) {
```

Comment:
I still don't understand why these live in two events when they represent the same user action.

Reply:
its the way data team set up the events one for each quiz rather then for the whole lesson